OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -1518,7 +1710,10 @@ public static DreamValue NativeProc_shutdown(DreamObject instance, DreamObject u
         [DreamProcParameter("PlainText", Type = DreamValueType.String)]
         [DreamProcParameter("format", Type = DreamValueType.Float, DefaultValue = 0)]
         public static DreamValue NativeProc_url_encode(DreamObject instance, DreamObject usr, DreamProcArguments arguments) {
-            string plainText = arguments.GetArgument(0, "PlainText").GetValueAsString();
+            if (!arguments.GetArgument(0, "PlainText").TryGetValueAsString(out var plainText))
+            {
+                return new DreamValue("");
+            }
```

Comment:
`url_encode()` turns any value given into a string
```suggestion
            string plainText = arguments.GetArgument(0, "PlainText").Stringify();
```